golang/cosmos/app/app.go

```diff
@@ -751,6 +751,22 @@ func NewAgoricApp(
 		upgrade10Handler(app, upgradeNameTest),
 	)
 
+	/* REVIEWER: confirm that this isn't needed
```

Comment:
I have no idea what this is supposed to be about, so I'll let other reviewers review.

Reply:
if this is aligned to the rho there were no store changes and we can skip this part

Reply:
No, not rho. We're still on cosmos-sdk 0.45.x. But the only change from upgrade-9 (pismoC) are the state sync changes which I believe don't need upgrade support. Is that correct, @mhofman ?

Reply:
Correct, state-sync is a new feature so it doesn't need any migration.